site/_posts/2017-01-03-last-week-in-manageiq-a-brand-new-euwe.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Last Two Weeks in ManageIQ: A brand new Euwe"
+author: Julian Cheal
+date: 2017-01-03
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+As the end of 2016 grew closer, and the Winter Solstice brought with it either the longest night or shortest night, depending on which hemisphere you frequent, ManageIQ cut its latest release Euwe. You can download it [here][1] or clone it from Github [here][2] Happy New Year!
+
+## Featured
+
+Over the last two weeks amongst all the New Year's celebrations there were [70 pull requests merged][3]! These pull requests (PRs) were comprised of [229 commits][4]. Why not start your New Year by contributing to Open Source, just checkout our [issue tracker][5] to see something that tickles your fancy, or join our chat on [Gitter][6] to find out more.
```

Comment:
Clearly, there should be a British slang verb meaning "to get more information."